File: represent/bench/java_flat/SpecGenBench_0349__SpecGenBench__common__PowerOfFourBranch__PowerOfFourBranch.java
class PowerOfFourBranch {
    public boolean isPowerOfFour(int n) {
        if(n <= 0) {
            return false;
        }
        if((n & (n - 1)) != 0) {
            return false;
        }
        if((n & 0xaaaaaaaa) != 0) {
            return false;
        }
        return true;
    }
}

File: represent/bench/c_flat/SpecGenBench_0349__SpecGenBench__common__PowerOfFourBranch__PowerOfFourBranch.c
int isPowerOfFour(int n);

int isPowerOfFour(int n) {

        if(n <= 0) {
            return 0;
        }
        if((n & (n - 1)) != 0) {
            return 0;
        }
        if((n & 0xaaaaaaaa) != 0) {
            return 0;
        }
        return 1;
}
